aten/src/ATen/autocast_mode.cpp

```diff
@@ -360,6 +360,10 @@ Therefore, for the moment, this is all copy pasted in from VariableTypeEverythin
   m.impl(REGISTER_NAME, \
     &WrapFunction<CastPolicy::POLICY, SIGNATURE, SIGNATURE, &FUNC>::type::call);
 
+#define KERNEL_HACKY_WRAPPER_FOR_LEGACY_SIGNATURES(FUNC, REGISTER_NAME, SIGNATURE, LEGACY_SIGNATURE, POLICY) \
```

Comment:
Can I pitch `KERNEL_LEGACY_SIGNATURE`? I think that conveys the appropriate level of contingency, without being quite so much of an eyesore (intended, I know! :). It'd be one thing if these were all go away in a week, but it's not quite that ephemeral.